http/http_test.go

```diff
@@ -0,0 +1,50 @@
+package http
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"slices"
+	"testing"
+)
+
+func TestGetJSON(t *testing.T) {
+	t.Run("ABC", func(t *testing.T) {
+		t.Parallel()
+
+		// testserver
+		ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			fmt.Fprintln(w, []byte{0, 1})
```

Comment:
you should write a `[]byte` to the response writer that corresponds to the  wanted return body of the http request.

Here you're sending back a literal 0 and a literal 1, not `"0"` and `"1"`.

Reply:
See https://github.com/nobe4/action-ln/blob/main/internal/github/branch_test.go#L22-L23 as an example